src/PreferencesDialog.ui

```diff
@@ -1866,6 +1866,16 @@ Can be set to 0 for disabling completion.</string>
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QCheckBox" name="checkSqleanExtensionAutoload">
+         <property name="toolTip">
+          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Automatically load the 'nalgeon/sqlean' extension at program startup.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
How fancy can the tooltip be?

If wonder if we should add a second line to it, saying something like:

    For more information about this extension, please see https://github.com/nalgeon/sqlean

Reply:
Nice idea, resolved via [47a036c](https://github.com/sqlitebrowser/sqlitebrowser/pull/3541/commits/47a036c47f7f9b2f09ecf7832732a45e464b1029) :)

Reply:
Should we add it to our `AboutDialog` as well?

Reply:
Good idea.  Yep, lets do that. :smile:

Reply:
```
We use the library in nalgeon/sqlean for SQLite extensions support.
This library is licensed under the MIT license, see the following for more information:
https://github.com/nalgeon/sqlean
```

Here's a draft, how about this?

Reply:
For the first line, we can slightly simplify it to:

    We use the nalgeon/sqlean library for SQLite extensions support.

The second line is fine as it is. :smile:

Reply:
Thanks, I'll make the change.